clang/lib/Parse/ParseExpr.cpp

```diff
@@ -1018,6 +1018,7 @@ ExprResult Parser::ParseCastExpression(CastParseKind ParseKind,
 
     // primary-expression
   case tok::numeric_constant:
```

Comment:
Can you adapt the couple other places where we parse a numeric constant?

Reply:
Could you please elaborate which other places you meant? I briefly examined all mentions of `tok::numeric_constant` and `ActOnNumericConstant` in `lib/Parse` and modifying any of these doesn't seem to be reasonable for `#embed` support.

Reply:
ping @cor3ntin

Reply:
These cases

```cpp
[[clang::availability(
   #embed "smth.txt"
)]] void f();

struct S {
   virtual void f() =
#embed "zero.bin"
    ;
};
```

However, it's a bit ambiguous whether these should be supported at all in the proposed C++ wording for embed. I sent a mail to the C++ committee

It's probably fine to ignore that for now, the patch is a great improvement as it is